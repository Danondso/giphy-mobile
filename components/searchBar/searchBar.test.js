import React from 'react';
import SearchBar from './searchBar';
import renderer from 'react-test-renderer';

test('renders SearchInput correctly', () => {
    const tree = renderer
      .create(<SearchBar />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });