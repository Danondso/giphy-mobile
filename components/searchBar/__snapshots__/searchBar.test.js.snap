// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders SearchInput correctly 1`] = `
<TextInput
  allowFontScaling={true}
  rejectResponderTermination={true}
  style={
    Object {
      "backgroundColor": "#E6EBE0",
      "borderColor": "#000",
      "borderRadius": 4,
      "borderWidth": 2,
      "marginBottom": 25,
      "marginTop": 100,
      "width": 200,
    }
  }
  underlineColorAndroid="transparent"
/>
`;
